src/components/Details/VmDetails/VmDetails.js

```diff
@@ -117,13 +132,17 @@ export class VmDetails extends React.Component {
       LoadingComponent,
       k8sGet,
     } = this.props;
+    const vmIsOff = this.isVmOff(vm, launcherPod, importerPod, migration);
     const nodeName = getNodeName(launcherPod);
     const ipAddresses = getVmiIpAddresses(vmi);
     const template = getVmTemplate(vm);
     const editButton = (
-      <Button disabled={this.state.updating} onClick={() => this.toggleEditing(true)}>
-        Edit
-      </Button>
+      <Fragment>
+        {!vmIsOff && <div className="kubevirt-vm-details__edit-warning">Please turn off the VM before editing.</div>}
```

Comment:
is this rendered as a text? Wouldn't it be better to use some warning/info icon?

Reply:
info icon sounds good!

Reply:
![edit_dis](https://user-images.githubusercontent.com/2078045/51037925-fc6e6a00-15b1-11e9-9421-f3a4b9aa54a5.png)

added info icon with text